### PR TITLE
implementing rating table 

### PR DESCRIPTION
**Pull request by frankyToast**
Hello! It would be nice if I could get your rating table so that I could see my changes reflect appropriately. Thank you!

### CODE REVIEW

**Comment by tylerJordan223** [comment]
Looks good!

**Review by tylerJordan223 — APPROVED** [review]
Looks good, hope it helps with tests.

File: tests/unit/test_get_all_movies.py
Comment (frankyToast) [inline tests/unit/test_get_all_movies.py:1]: You can also test to see if it works when you delete a movie as well

File: tests/e2e/test_all_movies_page.py
Comment (frankyToast) [inline tests/e2e/test_all_movies_page.py:1]: can also test if you delete a movie that it does not show

**Review by frankyToast — COMMENTED** [review]
Should go back and test that if a user removes a movie that it does not show show

**Pull request merged**